azuread/table_azuread_sign_in_report.go

```diff
@@ -106,9 +106,9 @@ func listAdSignInReports(ctx context.Context, d *plugin.QueryData, _ *plugin.Hyd
 	}
 
 	err = pageIterator.Iterate(ctx, func(pageItem interface{}) bool {
-		signIn := pageItem.(models.SignInable)
-
-		d.StreamListItem(ctx, &ADSignInReportInfo{signIn})
+		if signIn, ok := pageItem.(models.SignInable); ok {
```

Comment:
Can you please add a comment mentioning the other model type we can get here? It will help explain why we're doing a type check here